### DISCUSSION TITLE
Abell.js Blog/Portfolio Template

### DISCUSSION THREAD

**Discussion by haaarshsingh** [abelljs/abell#123] [Show your Abell Projects Here]
👋 Hey all,

I've made a new blog / portfolio template using Abell.js called "Calvera"

![cover](https://github.com/harshhhdev/calvera/blob/master/src/static-assets/Calvera.png?raw=true)
![image](https://user-images.githubusercontent.com/69592270/127592798-68722574-771d-4878-8426-2433eee350b0.png)

You can view a [demo](https://harshhhdev.github.io/calvera)

**Comment by haaarshsingh** [comment]
Here's a video demonstration if you need:

https://cdn.discordapp.com/attachments/706385518539309100/862211680720388156/2021-07-07_02-00-28.mp4

**Reply by saurabhdaware** [reply]
Looks great Harsh!
